x-pack/plugin/ilm/src/test/java/org/elasticsearch/xpack/indexlifecycle/IndexLifecycleInitialisationTests.java

```diff
@@ -72,7 +72,7 @@
 import static org.hamcrest.core.IsNull.nullValue;
 
 @ESIntegTestCase.ClusterScope(scope = Scope.TEST, numDataNodes = 0)
-public class IndexLifecycleInitialisationIT extends ESIntegTestCase {
+public class IndexLifecycleInitialisationTests extends ESIntegTestCase {
```

Comment:
This is necessary to run this class with `integTest.enabled = false` - several other plugins also do this.

Reply:
It is how other plugins do it, yeah. There isn't a clear definition of "correct" here.